paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -133,12 +133,14 @@ export interface WriteIdsAndPrefsOptions extends Options {
   redirectUrl?: URL;
 }
 
+export interface CreateSeedOptions extends Options {
+  callback?: Function;
```

Comment:
Can we define the signature of the callback? From I read, it should be something like that:
```
callback?: (seed: Seed) =>void;
```